addons/edi_notifier/models/notifier.py

```diff
@@ -167,13 +178,28 @@ def action_view_cron(self):
         }
         return action
 
+    @api.multi
+    def check_disbale_edi_notifications(self):
+        """Check safety config parameter is present and return True if it's enable, otherwise False."""
+        self.ensure_one()
+        if self.safety:
+            section, _sep, key = self.safety.rpartition('.')
+            if config.get_misc(section or self._name, key):
```

Comment:
This will disable if the safety catch is in the config file, shouldn't it be the other way around ?

Reply:
I'm not sure I follow what you mean here.
If the safety catch is set in the config file and it is set to True, this will disable the notifications, vice versa if set to False.
It will only disable if the safety catch is set through the interface.